tests/data/test.rdf

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This will produce relative URIs, which allows for testing of `base_iri` arg of load/bulk_load